recipes/default.rb

```diff
@@ -19,7 +19,9 @@
 
 include_recipe 'chef_handler'
 
-chef_gem 'hipchat'
+chef_gem 'hipchat' do
+  compile_time false if respond_to?(:compile_time)
```

Comment:
This should be true to keep the existing behavior or we can not report on compile time errors